src/commands/diffs.ts

```diff
@@ -15,7 +15,7 @@ async function selectForCompareCommand(item: any) {
     return;
   }
   const uri: vscode.Uri = convertItemToUri(item);
-  logger.info("Selected item for compare", uri);
+  logger.debug("Selected item for compare", uri);
```

Comment:
Unrelated to the Schema Registry bits of this PR, just noticed these at `info` level during clicktesting and thought it worth adding a little more cleanup